pkg/client/token_source.go

```diff
@@ -151,7 +151,9 @@ func (c *c1TokenSource) Token() (*oauth2.Token, error) {
 
 	tokenHost := c.tokenHost
 	if envHost, ok := os.LookupEnv("CONE_API_ENDPOINT"); ok {
```

Comment:
Do the same as what I put before also these if statements are backwards.

Use if it's NOT empty, not if it is